ci/azure-pipelines.yml

```diff
@@ -31,6 +31,17 @@ jobs:
       - script: bash <(curl https://codecov.io/bash)
         displayName: Upload coverage to Codecov
 
+  - job: BDDTest
+    pool:
+      vmImage: ubuntu-16.04
+    timeoutInMinutes: 30
+    steps:
+      - template: azp-dependencies.yml
+      - checkout: self
+      - script: make bdd-test
+                echo $DOCKER_PASSWORD | docker login docker.pkg.github.com --username $DOCKER_USER --password-stdin
```

Comment:
this line should be before make bdd-test not after

Reply:
done